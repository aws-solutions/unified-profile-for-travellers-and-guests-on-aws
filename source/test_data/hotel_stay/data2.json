{"objectVersion":3,"modelVersion":"1.0","id":"4WRSMN2GGX","bookingId":"FSGQKD11KU","guestId":"0227613903","lastUpdatedOn":"2022-05-13T00:01:48.780858187Z","createdOn":"2022-05-10T16:40:16.485981823Z","lastUpdatedBy":"Omer Mann","createdBy":"Jakob Rice","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"firstName":"Dave","lastName":"Funk","email":"augustaglover@kling.com","phone":"69749.816.2516","startDate":"2022-05-07","hotelCode":"PAR08","revenue":[{"type":"Room Charge 2022-05-07","description":"King Room with Balcony ( Super Saver rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":1191.2657,"date":"2022-05-07T00:00:00Z"},{"type":"Room Charge 2022-05-08","description":"King Room with Balcony ( Super Saver rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":1111.6967,"date":"2022-05-08T00:00:00Z"},{"type":"Room Charge 2022-05-09","description":"King Room with Balcony ( Super Saver rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":477.43668,"date":"2022-05-09T00:00:00Z"},{"type":"Room Charge 2022-05-10","description":"King Room with Balcony ( Super Saver rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":644.8661,"date":"2022-05-10T00:00:00Z"},{"type":"Room Charge 2022-05-11","description":"King Room with Balcony ( Super Saver rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":1081.671,"date":"2022-05-11T00:00:00Z"},{"type":"Room Charge 2022-05-12","description":"King Room with Balcony ( Super Saver rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":467.39627,"date":"2022-05-12T00:00:00Z"},{"type":"Room Charge 2022-05-13","description":"King Room with Balcony ( Super Saver rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":725.16656,"date":"2022-05-13T00:00:00Z"},{"type":"Room Charge 2022-05-14","description":"King Room with Balcony ( Super Saver rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":914.9351,"date":"2022-05-14T00:00:00Z"},{"type":"restaurant","description":"","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":48.877514,"date":"2022-05-09T03:41:44.754553442Z"},{"type":"just-walk-out-purchase","description":"","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":81.06377,"date":"2022-05-09T19:50:11.125804515Z"},{"type":"just-walk-out-purchase","description":"","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":22.115717,"date":"2022-05-13T16:26:52.44394501Z"},{"type":"Room Charge 2022-05-07","description":"Suite with  ( Xmas special rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":3584.511,"date":"2022-05-07T00:00:00Z"},{"type":"Room Charge 2022-05-07","description":"Suite with  ( Xmas special rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":2392.1147,"date":"2022-05-07T00:00:00Z"},{"type":"Room Charge 2022-05-07","description":"Suite with Balcony ( Xmas special rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":3916.9956,"date":"2022-05-07T00:00:00Z"},{"type":"Room Charge 2022-05-07","description":"Doubel room with  ( Best avaialble rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":1478.0857,"date":"2022-05-07T00:00:00Z"},{"type":"Room Charge 2022-05-07","description":"Doubel room with Sea View ( Super Saver rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":750.92706,"date":"2022-05-07T00:00:00Z"},{"type":"Room Charge 2022-05-07","description":"King Room with Mini bar ( Xmas special rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":1321.3273,"date":"2022-05-07T00:00:00Z"},{"type":"Room Charge 2022-05-08","description":"Suite with  ( Xmas special rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":3068.672,"date":"2022-05-08T00:00:00Z"},{"type":"Room Charge 2022-05-08","description":"Suite with  ( Xmas special rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":1369.9854,"date":"2022-05-08T00:00:00Z"},{"type":"Room Charge 2022-05-08","description":"Suite with Balcony ( Xmas special rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":1585.0684,"date":"2022-05-08T00:00:00Z"},{"type":"Room Charge 2022-05-08","description":"Doubel room with  ( Best avaialble rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":1453.7881,"date":"2022-05-08T00:00:00Z"},{"type":"Room Charge 2022-05-08","description":"Doubel room with Sea View ( Super Saver rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":464.58643,"date":"2022-05-08T00:00:00Z"},{"type":"Room Charge 2022-05-08","description":"King Room with Mini bar ( Xmas special rate)","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":957.6085,"date":"2022-05-08T00:00:00Z"},{"type":"sodas","description":"","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":92.89345,"date":"2022-05-10T15:48:00.570819263Z"},{"type":"restaurant","description":"","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":59.609196,"date":"2022-05-07T06:03:37.16269454Z"},{"type":"restaurant","description":"","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":23.197788,"date":"2022-05-12T04:15:00.864369088Z"},{"type":"restaurant","description":"","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":10.809075,"date":"2022-05-07T16:49:25.413709127Z"},{"type":"restaurant","description":"","currency":{"code":"SGD","name":"Singapore Dollar","symbol":""},"amount":96.8326,"date":"2022-05-14T01:52:16.048126121Z"}]}