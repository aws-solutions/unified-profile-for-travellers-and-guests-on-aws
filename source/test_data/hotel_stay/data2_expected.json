{"hotel_stay_revenue_items": [{"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-07", "description": "King Room with Balcony ( Super Saver rate)", "amount": 1191.2657, "date": "2022-05-07T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-08", "description": "King Room with Balcony ( Super Saver rate)", "amount": 1111.6967, "date": "2022-05-08T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-09", "description": "King Room with Balcony ( Super Saver rate)", "amount": 477.43668, "date": "2022-05-09T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-10", "description": "King Room with Balcony ( Super Saver rate)", "amount": 644.8661, "date": "2022-05-10T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-11", "description": "King Room with Balcony ( Super Saver rate)", "amount": 1081.671, "date": "2022-05-11T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-12", "description": "King Room with Balcony ( Super Saver rate)", "amount": 467.39627, "date": "2022-05-12T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-13", "description": "King Room with Balcony ( Super Saver rate)", "amount": 725.16656, "date": "2022-05-13T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-14", "description": "King Room with Balcony ( Super Saver rate)", "amount": 914.9351, "date": "2022-05-14T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "restaurant", "description": "", "amount": 48.877514, "date": "2022-05-09T03:41:44.754553442Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "just-walk-out-purchase", "description": "", "amount": 81.06377, "date": "2022-05-09T19:50:11.125804515Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "just-walk-out-purchase", "description": "", "amount": 22.115717, "date": "2022-05-13T16:26:52.44394501Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-07", "description": "Suite with  ( Xmas special rate)", "amount": 3584.511, "date": "2022-05-07T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-07", "description": "Suite with  ( Xmas special rate)", "amount": 2392.1147, "date": "2022-05-07T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-07", "description": "Suite with Balcony ( Xmas special rate)", "amount": 3916.9956, "date": "2022-05-07T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-07", "description": "Doubel room with  ( Best avaialble rate)", "amount": 1478.0857, "date": "2022-05-07T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-07", "description": "Doubel room with Sea View ( Super Saver rate)", "amount": 750.92706, "date": "2022-05-07T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-07", "description": "King Room with Mini bar ( Xmas special rate)", "amount": 1321.3273, "date": "2022-05-07T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-08", "description": "Suite with  ( Xmas special rate)", "amount": 3068.672, "date": "2022-05-08T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-08", "description": "Suite with  ( Xmas special rate)", "amount": 1369.9854, "date": "2022-05-08T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-08", "description": "Suite with Balcony ( Xmas special rate)", "amount": 1585.0684, "date": "2022-05-08T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-08", "description": "Doubel room with  ( Best avaialble rate)", "amount": 1453.7881, "date": "2022-05-08T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-08", "description": "Doubel room with Sea View ( Super Saver rate)", "amount": 464.58643, "date": "2022-05-08T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "Room Charge 2022-05-08", "description": "King Room with Mini bar ( Xmas special rate)", "amount": 957.6085, "date": "2022-05-08T00:00:00Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "sodas", "description": "", "amount": 92.89345, "date": "2022-05-10T15:48:00.570819263Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "restaurant", "description": "", "amount": 59.609196, "date": "2022-05-07T06:03:37.16269454Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "restaurant", "description": "", "amount": 23.197788, "date": "2022-05-12T04:15:00.864369088Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "restaurant", "description": "", "amount": 10.809075, "date": "2022-05-07T16:49:25.413709127Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}, {"model_version": "1.0", "object_type": "hotel_stay_revenue", "last_updated": "2022-05-13T00:01:48.780858187Z", "created_on": "2022-05-10T16:40:16.485981823Z", "last_updated_by": "Omer Mann", "created_by": "Jakob Rice", "id": "4WRSMN2GGX", "booking_id": "FSGQKD11KU", "traveller_id": "0227613903", "currency_code": "SGD", "currency_name": "Singapore Dollar", "currency_symbol": "", "first_name": "Dave", "last_name": "Funk", "email": "augustaglover@kling.com", "phone": "69749.816.2516", "start_date": "2022-05-07", "hotel_code": "PAR08", "type": "restaurant", "description": "", "amount": 96.8326, "date": "2022-05-14T01:52:16.048126121Z", "last_updated_partition": "2022/05/13/00/01/48/780858187Z"}]}